CmisSync.Lib/Sync/CrawlStrategy.cs

```diff
@@ -220,7 +220,7 @@ private void CrawlRemote(IFolder remoteFolder, string localFolder, IList<string>
                     }
                     else
                     {
-                        Logger.Debug("Unknown object type: " + cmisObject.ObjectType.DisplayName
+                        Logger.Warn("Unknown object type: " + cmisObject.ObjectType.DisplayName
```

Comment:
No, it is not. the idea is to catch these exotic types and handle them (maybe simply ignoring them because we know they are not useful). Most of the types should derive from Folder or Document, and others should be few (like links). At least this is in Alfresco; there are other implementations that differ?